.teamcity/_self/projects/WebApp.kt

```diff
@@ -766,6 +766,9 @@ object RunCalypsoPlaywrightE2eTests : BuildType({
 				# Decrypt config
 				openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%CONFIG_E2E_ENCRYPTION_KEY%"
 
+				# Compile TypeScript
+				yarn build
```

Comment:
nit: This should be part of `"Prepare environment"`

Reply:
You're right. Will move it in next commit.